ocdaction/tasks/admin.py

```diff
@@ -1,11 +1,17 @@
 from django.contrib import admin
-from .models import Task
+from .models import Task, AnxietyScore
 
 
 class TaskAdmin(admin.ModelAdmin):
     list_select_related = True
     list_display = ('taskname', 'user')
     search_fields = ['taskname']
 
```

Comment:
please include 2 empty lines before the next Class, if you are using Sublime or similar  please install a flake8 linter to help you with the correct spacing etc. http://flake8.pycqa.org/en/latest/